internal/driver/config.go

```diff
@@ -65,7 +66,6 @@ func (c *ServiceConfig) UpdateFromRaw(rawConfig interface{}) bool {
 }
 
 // GetCameraXAddr returns the Address:Port of the camera from the Onvif protocol properties
-// todo: unit test!
```

Comment:
Is this tested indirectly via your other tests?

Reply:
I ran into this while stepping through some of the unit tests. It looks like it has a full test dedicated to it, so I just thought I would remove the outdated comment.

Reply:
It is fine as long as it is outdated. I wanted to make sure the todo was done before deleting it.